content/en/database_monitoring/setup_mongodb/amazon_documentdb.md

```diff
@@ -0,0 +1,200 @@
+
+{{< tabs >}}
+{{% tab "Replica Set" %}}
+
+In a Mongo shell, authenticate to the primary node of the replica set, create a read-only user for the Datadog Agent in the `admin` database, and grant the required permissions:
+
+{{< code-block lang="shell" >}}
+
+# Authenticate as the admin user.
+
+use admin
+db.auth("admin", "<YOUR_AMAZON_DOCUMENTDB_ADMIN_PASSWORD>")
+
+# Create the user for the Datadog Agent.
+
+db.createUser({
+"user": "datadog",
+"pwd": "<UNIQUE_PASSWORD>",
+"roles": [
+{ role: "read", db: "admin" },
+{ role: "read", db: "local" },
+{ role: "clusterMonitor", db: "admin" }
+]
+})
+{{< /code-block >}}
+
+Grant additional permissions to the `datadog` user in the databases you want to monitor:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "read", db: "mydatabase" },
+{ role: "read", db: "myotherdatabase" }
+])
+{{< /code-block >}}
+
+Alternatively, you can grant `readAnyDatabase` role to the `datadog` user in the `admin` database to monitor all databases:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "readAnyDatabase", db: "admin" }
+])
+{{< /code-block >}}
+
+{{% /tab %}}
+{{% tab "Sharded Cluster" %}}
+
+1. For each shard in your cluster, connect to the primary node of the shard, create a read-only user for the Datadog Agent in the `admin` database, and grant the required permissions:
+
+{{< code-block lang="shell" >}}
+
+# Authenticate as the admin user.
+
+use admin
+db.auth("admin", "<YOUR_AMAZON_DOCUMENTDB_ADMIN_PASSWORD>")
+
+# Create the user for the Datadog Agent.
+
+db.createUser({
+"user": "datadog",
+"pwd": "<UNIQUE_PASSWORD>",
+"roles": [
+{ role: "read", db: "admin" },
+{ role: "read", db: "local" },
+{ role: "clusterMonitor", db: "admin" }
+]
+})
+{{< /code-block >}}
+
+Grant additional permissions to the `datadog` user in the databases you want to monitor:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "read", db: "mydatabase" },
+{ role: "read", db: "myotherdatabase" }
+])
+{{< /code-block >}}
+
+Alternatively, you can grant the `readAnyDatabase` role to the `datadog` user in the `admin` database to monitor all databases:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "readAnyDatabase", db: "admin" }
+])
+{{< /code-block >}}
+
+2. Follow the same steps and create the same user from a `mongos` proxy. This action creates the local user in the config servers and allows direct connection.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Securely store your password
+
+{{% dbm-secret %}}
+
+### Install and configure the Agent
+
+To monitor your Amazon DocumentDB Cluster, you must install and configure the Datadog Agent on a host that can [remotely access][1] your Amazon DocumentDB Cluster. This host can be a Linux host, a Docker container, or a Kubernetes pod.
+
+<div class="alert alert-info">Amazon DocumentDB Elastic clusters are not supported because they only expose the cluster (mongos) endpoints.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Amazon DocumentDB Elastic Clusters is not supported because it only exposes the cluster (<code>mongos</code>) endpoints.</div>
```